src/textual/_worker_manager.py

```diff
@@ -91,7 +91,7 @@ def _new_worker(
         start: bool = True,
         exclusive: bool = False,
         thread: bool = False,
-    ) -> Worker:
+    ) -> Worker[Any]:
```

Comment:
Looking at this, at line 84, and line 110, shouldn't all of these be connected?
`WorkType` is a generic with `ResultType` as a variable, which looks like it should match the generic argument here and in line 110.

Reply:
Ah yes, I missed that `WorkType` was a generic type itself, I'll add the missing type parameter to it and update the signature